pulsar-broker/src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java

```diff
@@ -330,9 +343,6 @@ private void refreshTopicPoliciesCache(Message<PulsarEvent> msg) {
                 case UPDATE:
                     policiesCache.put(topicName, event.getPolicies());
                     break;
-                case DELETE:
```

Comment:
what about old data present in existing clusters ?

Reply:
Good catch, maybe we could call the delete method to publish a message with a null value in branch `DELETE`?

Reply:
Indeed, for the compatibility, this code cannot be deleted now, I will add a comment here.
But Topic's existing old data seems to need to provide additional tools to delete it.

Reply:
When recovering the topic policies, the reader will read messages from the earliest position, it could read the `DELETE` type Pulsar event, does it handle the `DELETE` event messages at this time?

Reply:
I added a compatibility adaptation